eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,291 +1,284 @@
-        print('ClinVar accession: ' + clinvarRecord.accession)
-        print(err)
-        print(json.dumps(ev_string))
-        sys.exit(1)
-    except efo_term.EFOTerm.IsObsoleteException as err:
-        print('Error: obsolete EFO term.')
-        print('Term: ' + ev_string.get_disease().efoid)
-        print(err)
-        print(json.dumps(ev_string))
-        sys.exit(1)
-
-    return n_evidence_strings_per_record
+                if allele_origin == 'germline':
+                    # todo look into if any of these arguments to ev strings can be removed and their use extracted out
+                    evidence_string = evidence_strings.CTTVGeneticsEvidenceString(record,
+                                                                                  report,
+                                                                                  trait,
+                                                                                  ensembl_gene_id)
+                elif allele_origin == 'somatic':
+                    evidence_string = evidence_strings.CTTVSomaticEvidenceString(record,
+                                                                                 report,
+                                                                                 trait,
+                                                                                 ensembl_gene_id)
+                report.add_evidence_string(record, evidence_string)
+                report.evidence_list.append(
+                    [record.clinvarRecord.accession, record.rs, ','.join(trait.clinvar_trait_list),
+                     ','.join(trait.efo_list)])
+                report.counters["n_valid_rs_and_nsv"] += (record.clinvarRecord.get_nsv(mappings.rcv_to_nsv) is not None)
+                report.counters["n_more_than_one_efo_term"] += (len(trait.efo_list) > 1)
+                report.traits.update(set(trait.efo_list))
+                report.ensembl_gene_id_uris.add(evidence_strings.get_ensembl_gene_id_uri(ensembl_gene_id))
+
+                record.n_ev_strings_per_record += 1
+
+        if record.n_ev_strings_per_record > 0:
+            report.counters["n_processed_clinvar_records"] += 1
+            if record.n_ev_strings_per_record > 1:
+                report.counters["n_multiple_evidence_strings"] += 1
+
+    return report
+
+
+def get_mappings(efo_mapping_file, ignore_terms_file, adapt_terms_file, snp_2_gene_file, variant_summary_file):
+    mappings = SimpleNamespace()
+    mappings.trait_2_efo, mappings.unavailable_efo_dict = load_efo_mapping(efo_mapping_file, ignore_terms_file,
+                                                                           adapt_terms_file)
+
+    mappings.consequence_type_dict = consequence_type.process_consequence_type_file(snp_2_gene_file)
+    mappings.rcv_to_rs, mappings.rcv_to_nsv = clinvar_record.get_rcv_to_rsnsv_mapping(variant_summary_file)
+
+    return mappings
+
+
+def create_record(cellbase_record, mappings, **kwargs):
```

Comment:
It's just in the tests, otherwise a cellbase_record is needed, and mappings need to be created.